packages/@react-aria/utils/src/inertValue.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export function inertValue(value: any) {
+  if (typeof React['use'] === 'function') {
```

Comment:
I think we should check React.version here instead, just to avoid `use` being pulled in by tree shaking when it isn't needed.